Tests/Unit/TestCase.php

```diff
@@ -15,35 +15,39 @@
  */
 abstract class TestCase extends FrameworkTestCase
 {
-    /**
-     * Prepares the test environment before each test.
-     */
-    protected function setUp() {
-        parent::setUp();
-        Monkey\setUp();
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp()
+	{
+		parent::setUp();
+		Monkey\setUp();
 
-        $this->setup_common_wp_stubs();
-    }
+		$this->setup_common_wp_stubs();
+	}
 
-    /**
-     * Cleans up the test environment after each test.
-     */
-    protected function tearDown()
-    {
-        Monkey\tearDown();
-        parent::tearDown();
-    }
+	/**
+	 * Cleans up the test environment after each test.
+	 */
+	protected function tearDown()
+	{
+		Monkey\tearDown();
+		parent::tearDown();
+	}
 
-    /**
-     * Set up the stubs for the common WordPress escaping and internationalization functions.
-     */
-    protected function setup_common_wp_stubs() {
-        // Common internationalization functions.
-        Monkey\Functions\stubs( array(
-            '__',
-            'esc_html__',
-            'esc_html_x',
-            'esc_attr_x',
-        ) );
-    }
+	//phpcs:disable
+	/**
+	 * Set up the stubs for the common WordPress escaping and internationalization functions.
+	 */
+	protected function setup_common_wp_stubs()
```

Comment:
@hellofromtonya I did phpcs:disable here because I didn't know why `setup_common_wp_stubs` isn't camel cased.

Reply:
Make it camelCase.  It's a copy from my other work, which uses WPCS 100%.